cmd/romana.k8s

```diff
@@ -0,0 +1,403 @@
+			continue
+		fi
+
+		# Write the data
+		printf "owner:%-32s\n" "$i"
+		printf "tiers: "
+		sep=""
+		while read -r tier_name _; do
+			printf "%s%s" "$sep" "$tier_name"
+			sep=", "
+		done < <(jq -r '.[].Name' <<< "$tier_list")
+		printf "\n"
+	done
+}
+
+show_owners() {
+	# Get the list of owners
+	if ! response=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}"); then
+		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		return
+	fi
+	# Determine the number of owners
+	num_owners=$(jq 'length' <<< "$response")
+	if (( num_owners == 0 )); then
+		echo "No owners found."
+		return
+	fi
+	# List the owners
+	printf "Listing %d owner(s)\n" "$num_owners"
+	while read -r owner _; do
+		printf "%-32s\n" "$owner"
+	done < <(jq -r '.[].Name' <<< "$response")
+}
+
+add_tier() {
+	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
+		echo "romana add-tier <owner-name> <segment-name>"
+		return
+	fi
+	if (( $# != 2 )); then
+		echo "ERROR: expected 2 parameters, received $#"
+		return
+	fi
+	local owner_name="$1" tier_name="$2"
+	if ! response=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}"); then
+		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		return
+	fi
+	read -r owner_id _ < <(jq -r --arg name "$owner_name" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
+
+	if ! [[ "$owner_id" ]]; then
+		echo "Unable to find owner id using owner ref '$owner_name'"
+		return
+	fi
+
+	post_data=$(printf '{"id": 0, "name": "%s"}' "$tier_name")
+	tier_path="$tenant_path/$owner_id/segments"
+	echo "Sending request to '${tenant_url}${tier_path}': '$post_data'"
+	curl -s -X POST -H "Content-Type: application/json" --data "$post_data" "${tenant_url}${tier_path}"; printf "\n"
+}
+
+add_policy() {
+	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
+		echo "romana add-policy <policy-file>"
+		return
+	fi
+	if (( $# != 1 )); then
+		echo "ERROR: expected 1 parameters, received $#"
+		return
+	fi
+	local policy_file="$1"
+	if ! [[ -f "$policy_file" ]]; then
+		echo "Unable to find file '$policy_file'"
+		return
+	fi
+	echo "Sending POST request to '${kube_apiserver_url}${romana_policy_path}': '@$policy_file'"
+	curl -s -X POST -H "Content-Type: application/json" --data "@$policy_file" "${kube_apiserver_url}${romana_policy_path}"; printf "\n"
+}
+
+remove_policy() {
+	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
+		echo "romana add-policy <policy-file>"
+		return
+	fi
+	if (( $# != 1 )); then
+		echo "ERROR: expected 1 parameters, received $#"
+		return
+	fi
+	local policy_name="$1"
+	echo "Sending DELETE request to '${kube_apiserver_url}${romana_policy_path}/$policy_name'"
+	curl -s -X DELETE "${kube_apiserver_url}${romana_policy_path}/$policy_name"; printf "\n"
+}
+
+show_policy() {
+	# Without arguments, show all the policys
+	if (( $# == 0 )); then
+		show_policies
+		return
+	fi
+
+	# Show usage.
+	# TODO: What if the policy's name is 'help' ?
```

Comment:
How about we do not allow the creation of a policy that's named "help"? Treat it as a reserved name?

Alternatively, use "--help".